<head>
<meta name="robots" content="nofollow" />
<meta name="generator" content="FreeBSD-CVSweb 3.0.6" />
<meta http-equiv="Content-Script-Type" content="text/javascript" />
<meta http-equiv="Content-Style-Type" content="text/css" />
<title>view</title>
<meta http-equiv='content-type' content='text/html; charset=iso-8859-1' >
<meta name='robots' content='nofollow' >
    <link rel="stylesheet" media="screen"
    href="http://www.FreeBSD.org/layout/css/fixed.css" type="text/css"
    title="Normal Text" >
    <link rel="alternate stylesheet" media="screen"
    href="http://www.FreeBSD.org/layout/css/fixed_large.css" type="text/css"
    title="Large Text" >
    <link rel="shortcut icon" href="../../favicon.ico" type="image/x-icon" />
    <link rel="apple-touch-icon" href="../../favicon.ico" type="image/x-icon" />

<script type="text/javascript" src="http://www.FreeBSD.org/layout/js/styleswitcher.js">
</script>

<link rel="search" type="application/opensearchdescription+xml" href="http://www.freebsd.org/search/opensearch/cvsweb.xml" title="FreeBSD cvsweb" />

</head>
<body>

    <div id="containerwrap">
      <div id="container">
        <span class="txtoffscreen"><a href="#content"
        title="Skip site navigation" accesskey="1">Skip site
        navigation</a> (1)</span><span class="txtoffscreen"><a
        href="#content" title="Skip section navigation"
        accesskey="2">Skip section navigation</a> (2)</span>

        <div id="headercontainer">
          <div id="header">
            <h2 class="blockhide">Header And Logo</h2>

            <div id="headerlogoleft">
              <a href="http://www.FreeBSD.org" title="FreeBSD"><img
              src="http://www.FreeBSD.org/layout/images/logo-red.png" width="457"
              height="75" alt="FreeBSD" /></a>
            </div>

            <div id="headerlogoright">
              <h2 class="blockhide">Peripheral Links</h2>

              <div id="searchnav">
                <ul id="searchnavlist">
                  <li>Text Size: <a href="#"
                  onkeypress="return false;"
                  onclick="setActiveStyleSheet('Normal Text'); return false;"
                   title="Normal Text Size">Normal</a> / <a
                  href="#" onkeypress="return false;"
                  onclick="setActiveStyleSheet('Large Text'); return false;"
                   title="Large Text Size">Large</a></li>

                  <li><a href="http://www.FreeBSD.org/donations/"
                  title="Donate">Donate</a></li>

                  <li class="last-child"><a href="http://www.FreeBSD.org/mailto.html"
                  title="Contact">Contact</a></li>
                </ul>
              </div>

              <div id="search">
                <form
                action="http://www.FreeBSD.org/cgi/search.cgi"
                method="get">
                  <div>
                    <h2 class="blockhide"><label
                    for="words">Search</label></h2>
                    <input type="hidden" name="max"
                    value="25" /><input type="hidden" name="source"
                    value="www" /><input id="words" name="words"
                    type="text" size="20" maxlength="255"
                    onfocus="if( this.value==this.defaultValue ) this.value='';"
                     value="Search" />&nbsp;<input id="submit"
                    name="submit" type="submit" value="Search" />
                  </div>
                </form>
              </div>
            </div>
          </div>

          <h2 class="blockhide">Site Navigation</h2>

	  <div id="MENU">
	    <ul class="first">
	      <li><a href="http://www.FreeBSD.org/">Home</a></li>
	    </ul>
	    <ul>
	      <li><a href="http://www.FreeBSD.org/about.html">About</a>
		<ul>
		  <li><a href="http://www.FreeBSD.org/projects/newbies.html">Introduction</a></li>
		  <li><a href="http://www.FreeBSD.org/features.html">Features</a></li>
		  <li><a href="http://www.FreeBSD.org/advocacy/">Advocacy</a></li>
		  <li><a href="http://www.FreeBSD.org/marketing/">Marketing</a></li>
		</ul>
	      </li>
	    </ul>
	    <ul>
	      <li><a href="http://www.FreeBSD.org/where.html">Get FreeBSD</a>
		<ul>
		  <li><a href="http://www.FreeBSD.org/releases/">Release Information</a></li>
		  <li><a href="http://www.FreeBSD.org/releng/">Release Engineering</a></li>
		</ul>
	      </li>
	    </ul>
	    <ul>
	      <li><a href="http://www.FreeBSD.org/docs.html">Documentation</a>
		<ul>
		  <li><a href="http://www.FreeBSD.org/doc/en_US.ISO8859-1/books/faq/">FAQ</a></li>
		  <li><a href="http://www.FreeBSD.org/doc/en_US.ISO8859-1/books/handbook/">Handbook</a></li>
		  <li><a href="http://www.FreeBSD.org/doc/en_US.ISO8859-1/books/porters-handbook">Porter's Handbook</a></li>
		  <li><a href="http://www.FreeBSD.org/doc/en_US.ISO8859-1/books/developers-handbook">Developer's Handbook</a></li>
		  <li><a href="http://www.FreeBSD.org/cgi/man.cgi">Manual Pages</a></li>
		</ul>
	      </li>
	    </ul>
	    <ul>
	      <li><a href="http://www.FreeBSD.org/community.html">Community</a>
		<ul>
		  <li><a href="http://www.FreeBSD.org/community/mailinglists.html">Mailing Lists</a></li>
		  <li><a href="http://forums.freebsd.org">Forums</a></li>
		  <li><a href="http://www.FreeBSD.org/usergroups.html">User Groups</a></li>
		  <li><a href="http://www.FreeBSD.org/events/events.html">Events</a></li>
		</ul>
	      </li>
	    </ul>
	    <ul>
	      <li><a href="http://www.FreeBSD.org/projects/index.html">Developers</a>
		<ul>
		  <li><a href="http://www.FreeBSD.org/projects/ideas/ideas.html">Project Ideas</a></li>
		  <li><a href="http://svn.FreeBSD.org/viewvc/base/">SVN Repository</a></li>
		  <li><a href="http://cvsweb.FreeBSD.org">CVS Repository</a></li>
		  <li><a href="http://p4web.FreeBSD.org">Perforce Repository</a></li>
		</ul>
	      </li>
	    </ul>
	    <ul>
	      <li><a href="http://www.FreeBSD.org/support.html">Support</a>
		<ul>
		  <li><a href="http://www.FreeBSD.org/commercial/commercial.html">Vendors</a></li>
		  <li><a href="http://security.FreeBSD.org/">Security Information</a></li>
		  <li><a href="http://www.FreeBSD.org/cgi/query-pr-summary.cgi">Bug Reports</a></li>
		  <li><a href="http://www.FreeBSD.org/send-pr.html">Submit Bug-report</a></li>
		</ul>
	      </li>
	    </ul>
	    <ul>
	      <li><a href="http://www.freebsdfoundation.org/">Foundation</a>
		<ul>
		  <li><a href="http://www.freebsdfoundation.org/donate/">Donate</a></li>
		</ul>
	      </li>
	    </ul>
	  </div> <!-- MENU -->
        </div>

	<div id="content">

<h1>view</h1>
<link rel="stylesheet" type="text/css" href="/layout/css/cvsweb.css" />
<body class="src">
 <table class="navigate-header" width="100%" summary="Navigation">
  <tr>
   <td><a href="./patch-src%3A%3Amain.c#rev1.1"><img src="/gifs/back.gif" alt="[BACK]" border="0" width="20" height="22" /></a><b>Return to <a href="./patch-src%3A%3Amain.c#rev1.1">patch-src::main.c</a> CVS log</b> <img src="/gifs/text.gif" alt="[TXT]" border="0" width="20" height="22" /></td>  <td style="text-align: right"><img src="/gifs/dir.gif" alt="[DIR]" border="0" width="20" height="22" /> <b>Up to  <a href="/cgi/cvsweb.cgi/#dirlist">[FreeBSD]</a> / <a href="/cgi/cvsweb.cgi/ports/#dirlist">ports</a> / <a href="/cgi/cvsweb.cgi/ports/audio/#dirlist">audio</a> / <a href="/cgi/cvsweb.cgi/ports/audio/grip/#dirlist">grip</a> / <a href="/cgi/cvsweb.cgi/ports/audio/grip/files/#dirlist">files</a></b></td>
 </tr>
</table>
<hr />
<div class="log-markup">
File:&nbsp;
 <a href="/cgi/cvsweb.cgi/#dirlist">[FreeBSD]</a> / <a href="/cgi/cvsweb.cgi/ports/#dirlist">ports</a> / <a href="/cgi/cvsweb.cgi/ports/audio/#dirlist">audio</a> / <a href="/cgi/cvsweb.cgi/ports/audio/grip/#dirlist">grip</a> / <a href="/cgi/cvsweb.cgi/ports/audio/grip/files/#dirlist">files</a> / <a href="/cgi/cvsweb.cgi/ports/audio/grip/files/patch-src%3A%3Amain.c">patch-src::main.c</a><br />
<a name="rev1.1"></a><a name="RELEASE_7_3_0"></a><a name="HEAD"></a><a name="MAIN"></a>
 Revision <b>1.1</b>: <a href="/cgi/cvsweb.cgi/~checkout~/ports/audio/grip/files/patch-src%3A%3Amain.c?rev=1.1;content-type=text%2Fplain" class="download-link">download</a> - view: <a href="patch-src%3A%3Amain.c?rev=1.1;content-type=text%2Fplain" class="display-link">text</a>, <a href="patch-src%3A%3Amain.c?annotate=1.1">annotated</a> - <a href="patch-src%3A%3Amain.c?r1=1.1#rev1.1">select&nbsp;for&nbsp;diffs</a><br />
<i>Sun Nov 15 12:26:31 2009 UTC</i> (3 months, 4 weeks ago) by <i>naddy</i><br />
Branches: <a href="./patch-src%3A%3Amain.c?only_with_tag=MAIN">MAIN</a><br />
CVS tags: <a href="./patch-src%3A%3Amain.c?only_with_tag=RELEASE_7_3_0">RELEASE_7_3_0</a>,
<a href="./patch-src%3A%3Amain.c?only_with_tag=HEAD">HEAD</a><br />
<pre class="log">
Fix the actual bug that caused translated message catalogs to be
displayed incorrectly, and remove the workaround.
</pre>
</div>
<hr /><pre>
--- src/main.c.orig
+++ src/main.c
@@ -163,7 +163,7 @@
 		     GNOME_PARAM_POPT_TABLE,options,
 		     GNOME_PROGRAM_STANDARD_PROPERTIES,NULL);
 
-  bind_textdomain_codeset(GETTEXT_PACKAGE, &quot;UTF8&quot;);
+  bind_textdomain_codeset(GETTEXT_PACKAGE, &quot;UTF-8&quot;);
 
   /* Session Management */
   
</pre>

	</div>
        <div id="footer">
          <a href="http://www.FreeBSD.org/copyright/">Legal Notices</a> | &copy; 1995-2010
          The FreeBSD Project. All rights reserved.<br />
	  <address><a href='http://www.FreeBSD.org/mailto.html'>www@FreeBSD.org</a><br />2008/09/23 16:31:45</address>
        </div>
      </div>
    </div>
  </body>
</html>
